plugins/modules/mysql_user.py

```diff
@@ -116,6 +116,14 @@
       - Used when I(state=present), ignored otherwise.
     type: dict
     version_added: '0.1.0'
+  account_locking:
+    description:
+      - Configure user accounts such that too many consecutive login failures cause temporary account locking. Provided since MySQL 8.0.19.
+      - "Available options are C(FAILED_LOGIN_ATTEMPTS: num), C(PASSWORD_LOCK_TIME: num | UNBOUNDED)."
```

Comment:
```suggestion
      - "Available options are C(FAILED_LOGIN_ATTEMPTS: num), C(PASSWORD_LOCK_TIME: num | UNBOUNDED)."
```
shouldn't we check the options and show users that they passed an incorrect key and / or value type? Would be great to provide an informative message to users where they make typos or something

Reply:
Hmmm, I should write a test for that. My assumption was that the server would complain if the option's value is out of range/invalid.

Reply:
yes, it will but, imo, graceful exit with an informative helpful error is better

Reply:
@Andersson007 I just checked and the MySQL server does return a very clear error message when these values are out of bounds. I do not see how adding our own check beforehand and calling `module.err` would make any functional difference. Am I missing something?

Reply:
MySQL, in their usual fashion, only provides a helpful message in a fraction of the cases, the rest default to the usual "you have an error on your SQL syntax". What a pain.

Reply:
if we can't parse this anyhow, ok